projects/addon-commerce/components/input-card-group/input-card-group.component.ts

```diff
@@ -198,6 +201,7 @@ export class TuiInputCardGroup
     @Input()
     public id = tuiInjectId();
 
+    /** @deprecated apparently "off" doesn't disable autocomplete */
     @Input()
     public autocomplete = this.options.autocomplete;
```

Comment:
User can still set `autocomplete="new-password"`

https://taiga-ui.dev/components/input#autofill

Reply:
(same comment for other similar deprecations)

Reply:
It was boolean before, so no. In separate inputs we can override it if we want because the input is externally accessible. In here, let's just keep it always on.